spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTMPeephole.scala

```diff
@@ -35,9 +36,16 @@ import scala.reflect.ClassTag
 @SerialVersionUID(- 7566757838561436619L)
 class LSTMPeephole[T : ClassTag] (
   val inputSize: Int,
-  val hiddenSize: Int)
+  val hiddenSize: Int,
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out? and nochange in javadoc?